### PR TITLE
EDSC-3445: Pulls geo utils code out of Earthdata Search

### PR DESCRIPTION
**Pull request by macrouch**

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        targets: {
+          node: '14',
+          esmodules: true,
+          ie: '10'
+        }
+      }
+    ]
+  ],
+  sourceType: 'unambiguous'
+  // ignore: [
```
Comment (trevorlang) [inline babel.config.js]: Did you want this comment in the code still?

**Review by trevorlang — APPROVED** [review]

File: babel.config.js (hunk 1 above)
Comment (macrouch) [reply babel.config.js]: :doh:

File: README.md [hunk 2]
```diff
@@ -1 +1,44 @@
-# edsc-geo
+# Earthdata Search Components: Geo Utils
+
+[![npm version](https://badge.fury.io/js/%40edsc%2Fechoforms.svg)](https://badge.fury.io/js/%40edsc%2Fgeo-utils)
```
Comment (abbottry) [inline README.md]: `https://badge.fury.io/js/%40edsc%2Fechoforms.svg` Wrong SVG.

File: README.md [hunk 3]
```diff
@@ -1 +1,44 @@
...
+![Build Status](https://github.com/nasa/edsc-geo/workflows/CI/badge.svg?branch=master)
+[![codecov](https://codecov.io/gh/nasa/edsc-geo/branch/master/graph/badge.svg?token=4d8wFDtAc0)](https://codecov.io/gh/nasa/edsc-geo)
```
Comment (abbottry) [inline README.md:5]: I think the branch is `main` not `master`, right?

File: babel.config.js (hunk 1 above)
Comment (abbottry) [inline babel.config.js]: Still want to ignore this?

File: jest.config.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  collectCoverage: true,
+  coveragePathIgnorePatterns: [
+    'package.json',
+    'package-lock.json'
+  ],
+  testPathIgnorePatterns: [
+    'mocks.js'
```
Comment (abbottry) [inline jest.config.js]: There aren't any `mocks.js` files, this can be removed

File: package.json [hunk 5]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@edsc/geo-utils",
+  "version": "1.0.0",
+  "description": "Geographical utilities used by Earthdata Search",
```
Comment (abbottry) [inline package.json:4]: But this is also now used by smart hand-offs right? Thinking we should strip out these references to Earthdata Search expect for the fact that it was pulled from that codebase.. maybe?

File: README.md (hunk 2 above)
Comment (macrouch) [reply README.md]: :doh:

File: README.md (hunk 3 above)
Comment (macrouch) [reply README.md:5]: This repo is really old, master is the branch here

File: babel.config.js (hunk 1 above)
Comment (macrouch) [reply babel.config.js]: Pretty sure I removed this after Trevor's comment. It moved into the build command

**Review by abbottry — APPROVED** [review]

**Pull request merged**